package.json

```diff
@@ -13,6 +13,8 @@
     "react-router-dom": "^5.3.0",
     "react-scripts": "4.0.3",
     "redux": "^4.1.1",
+    "redux-logger": "^3.0.6",
+    "uuid": "^8.3.2",
     "web-vitals": "^1.0.1"
   },
   "scripts": {
```

Comment:
* Kindly remove this `ESLINT_NO_DEV_ERRORS=true` from the start script as it makes the project difficult to start, if there  is a reason for the command kindly update your readme file with instructions on how to properly start the project 🙏

Reply:
There is a reason for it to be there. It prevents linting errors from breaking the server project, which is a _very_ annoying behavior.
I'm curious, though, what kind of problems does it give you? What system did you try to run the app on?